conf/messages

```diff
@@ -129,15 +133,18 @@ problemDeclaring.contact.link = contact the trusts helpline (opens in a new wind
 
 confirmation.title = Registration received
 confirmation.heading = Registration received
-confirmation.heading2 = Your reference is:
-confirmation.printsave = Print or save a copy of the estate’s declaration
+confirmation.heading2 = Your registration reference number is:
```

Comment:
is this change is part of this ticket?

Reply:
All the confirmation parts are from my other ticket, 6134. I branched from there instead of main :/ should I recommit the branch with just the changes for unauthorised?